packages/assets-controllers/src/TokenRatesController.ts

```diff
@@ -538,7 +538,7 @@ export class TokenRatesController extends StaticIntervalPollingControllerV1<
       (obj, [tokenAddress, tokenPrice]) => {
         return {
           ...obj,
-          [tokenAddress]: tokenPrice.value,
+          [tokenAddress]: tokenPrice?.value,
```

Comment:
One last thing: I think we don't need this question mark anymore, since we know that if a token price is present, it always has a price.

Reply:
Hmmm  Actually with adding the "Partial" to the type i had to add the question mark; i guess with Partial it will assume that the field is optional?

If i remove the Partial i added on the types, then i can remove that question mark; 

I think since we are totally removing the object price from the response; we can leave the types as they were before; meaning without adding that Partial<TokenPricesByTokenAddress<Hex, SupportedCurrency>> instead of TokenPricesByTokenAddress<Hex, SupportedCurrency>

Reply:
This is what I get for answering on my phone 😅 You're right! No, this is fine.